Marlin/language_en.h

```diff
@@ -462,6 +462,9 @@
 #ifndef MSG_HEATING
   #define MSG_HEATING                         _UxGT("Heating...")
 #endif
+#ifndef MSG_DHT_WARMING
+  #define MSG_DHT_WARMING                     _UxGT("Sensor Warming...")
+#endif
```

Comment:
Yes...I wanted to display "Sensor Warming..." during the delay. I am still trying to nail down how to create that screen.